funnel/assets/cypress/integration/07_addCFP.js

```diff
@@ -101,7 +125,7 @@ describe('Add labels to project', () => {
       .trigger('mouseover', { which: 1, force: true, view: window })
       .trigger('mousedown', { which: 1, force: true, view: window })
       .trigger('mousemove', {
-        pageX: 500,
+        pageX: 650,
         pageY: 550,
         force: true,
         view: window,
```

Comment:
Is there a better way to test for dragging? Can it be done by retrieving pageX and pageY from other elements?